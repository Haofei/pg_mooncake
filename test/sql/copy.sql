\getenv pwd PWD
\set csv_file '\'' :pwd '/results/copy.csv'  '\''

COPY (SELECT * FROM generate_series(1, 5)) TO :csv_file;

CREATE TABLE t (a INT) USING columnstore;
COPY t FROM :csv_file;
SELECT * FROM t;

COPY t TO :csv_file;
COPY (SELECT * FROM t) TO :csv_file;

DROP TABLE t;
