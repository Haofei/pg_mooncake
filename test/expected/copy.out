\set pwd `pwd`
\set csv_file '\'' :pwd '/results/copy.csv'  '\''
COPY (SELECT * FROM generate_series(1, 5)) TO :csv_file;
CREATE TABLE t (a int) USING columnstore;
COPY t FROM :csv_file;
SELECT * FROM t;
 a 
---
 1
 2
 3
 4
 5
(5 rows)

COPY t TO :csv_file;
COPY (SELECT * FROM t) TO :csv_file;
DROP TABLE t;
